### DISCUSSION TITLE
How lighthouse detect resource caching?

### DISCUSSION THREAD

**Discussion by inikonorov** [GoogleChrome/lighthouse#15036] [Q&A]
Hi everyone. My page has some static files (js, in generally), which are not cached by "cache-control" header. In Lighthouse report I see, that these resources are not cached (seems to be true, because I don't set "cache-control" header), but in the network I see, that the same resources are loaded from disk/memory cache. What's going on? How does it work?

**Comment by adamraine** [comment]
What URL are you testing on?

**Reply by inikonorov** [reply]
localhost
